RaveBusinessLogic/VBET.xml

```diff
@@ -1,145 +1,50 @@
-              <Node label="Full NHD HR+ Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="flow_lines" />
-              <Node label="Perennial Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdperrenial" />
-              <Node label="Intermittent Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdintermittent" />
-              <Node label="Ephemeral Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="line" symbology="nhdephemeral" />
-              <Node label="Network - Upstream Drainage Network" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOWLINES']" type="polygon" symbology="upstreamdrainagearea" />
-              <Node label="River Polygons (NHD Area)" xpath="Inputs/Geopackage/Layers/Vector[@id='FLOW_AREA']" type="polygon" symbology="nhdarea" />
-              <Node xpathlabel="Name" xpath="Inputs/Geopackage/Layers/Vector[@id='CATCHMENTS']" type="polygon" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='CHANNEL_BUFFER_RASTER']" type="raster" symbology="" />
-              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='FLOW_AREA_RASTER']" type="raster" symbology="" />
-            </Children>
-          </Node>
-          <Node label="Topography">
-            <Children collapsed='true'>
-              <Node label="Slope Analysis" xpath="Inputs/Raster[@id='SLOPE_RASTER']" type="raster" symbology="slope" transparency="40" />
-              <Node label="Height Above Nearest Drainage" xpath="Intermediates/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
-              <Node label="Digital Elevation Model" xpath="Inputs/Raster[@id='DEM']" type="raster" symbology="dem" transparency="40" id="dem" />
-              <Node xpathlabel="Name" xpath="Inputs/Raster[@id='HILLSHADE']" type="raster" symbology="" id="hillshade" />
+              <Node label="Estimated Active Channel" xpath="Geopackage/Layers/Vector[@id='VBET_100']" type="polygon" symbology="vbet100outline" id="vbet100outline"/>
+              <Node label="Estimated Active Floodplain" xpath="Geopackage/Layers/Vector[@id='VBET_70']" type="polygon" symbology="vbet70outline" id="vbet70outline"/>
+              <Node label="Estimated Inactive Floodplain" xpath="Geopackage/Layers/Vector[@id='VBET_50']" type="polygon" symbology="vbet50outline" id="vbet50outline"/>      
             </Children>
           </Node>
         </Children>
       </Node>
+      <Node label="Inputs" xpath="Inputs">
+          <Children>
+            <!-- <Node label="Channel Area" xpath="Geopackage/Layers/Vector[@id='CHANNEL_AREA']" type="polygon" id="channel_area" symbology="ChannelArea" /> -->
+            <Node label="NHD Flow Areas" xpath="Geopackage/Layers/Vector[@id='FLOW_AREAS']" type="polygon" symbology="nhdarea" />

```

Comment:
These are no longer necessarily "NHD flow areas". Change to "Channel_Area" as you have in line 25.

Reply:
@joewheaton Because the `channel_area_polyon` layer includes the NHD `flow_areas`, I'm going to axe this node altogether to prevent redundancy and/or misconstruing the NHD `flow_areas` layer as a VBET input.